src/plugins/saved_objects_management/public/management_section/object_view/components/intro.tsx

```diff
@@ -44,12 +44,12 @@ export const Intro = () => {
       iconType="alert"
       color="warning"
     >
-      <div>
+      <p>
```

Comment:
`<div></div>` should be replaced with `<p></p>`, in order to indicate a paragraph of text